plugin/pluginCore.js

```diff
@@ -25,9 +25,9 @@ exports.runPa11y = async function({ htmlFilePaths, build, testMode, debugMode })
   return flattenedResults;
 };
 
-const runPa11yOnFile = async function(htmlFilePath, build) {
+const runPa11yOnFile = async function(htmlFilePath, build, timeout) {
   try {
-    return await pa11y(htmlFilePath)
+    return await pa11y(htmlFilePath, {timeout, chromeLaunchConfig: {timeout}})
```

Comment:
Sets the timeout for both pa11y and puppeteer (chromeLaunchConfig). Both are 30 seconds by default, so only extending pa11y's just caused puppeteer to timeout instead

Reply:
The newer version of pa11y seems to have fixed this, so setting chromeLaunchConfig doesn't seem to be required anymore